Testscripts/Linux/SetupHbKernel.sh

```diff
@@ -127,13 +123,23 @@ function Main() {
 		cd
 
 		# Append the test log to the main log files.
-		cat /usr/src/linux/TestExecution.log >> TestExecution.log
-		cat /usr/src/linux/TestExecutionError.log >> TestExecutionError.log
+		if [ -f /usr/src/linux/TestExecution.log ]; then
+			cat /usr/src/linux/TestExecution.log >> TestExecution.log
```

Comment:
why not write TestExecution.log in $HOME or set $homeDir?

Reply:
Do you remember this TestExecution.log and Error.log file are generated by LogMsg/LogErr if the run-time directory is different from $HOME?